### ISSUE TITLE
BUG - you have to reset the headers

### ISSUE THREAD

**Issue by beshoo** [xosad/Insta-mass-account-creator#24]
Dear- you need to reset the headers , you are add them in array - but each time you need to reset all headers once you create the account.

```

Array
(
    [0] => Host: www.instagram.com
    [1] => Accept: */*
    [2] => Accept-Language: en-US,en;q=0.5
    [3] => Accept-Encoding: gzip, deflate, br
    [4] => X-Instagram-AJAX: 62d4dd0faf8e-hot
    [5] => X-IG-App-ID: 936619743392459
    [6] => X-IG-WWW-Claim: 0
    [7] => Content-Type: application/x-www-form-urlencoded
    [8] => X-Requested-With: XMLHttpRequest
    [9] => Origin: https://www.instagram.com
    [10] => Connection: keep-alive
    [11] => Referer: https://www.instagram.com/
    [12] => Pragma: no-cache
    [13] => Cache-Control: no-cache
    [14] => X-CSRFToken: aVjmnA3IU6qiPklzhx0f5j0HP5ZMtrbx
    [15] => Cookie: ig_cb=1; mid=XflzXgAAAAHHZVFj4UHx6QLtJR_j; csrftoken=aVjmnA3IU6qiPklzhx0f5j0HP5ZMtrbx; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
)
Could not create the account. Error: {"account_created": false, "errors": {"ip": ["The IP address you are using has been flagged as an open proxy. If you believe this to be incorrect, please visit http://help.instagram.com/"]}, "status": "ok", "error_type": "signup_block"}
Array
(
    [0] => Host: www.instagram.com
    [1] => Accept: */*
    [2] => Accept-Language: en-US,en;q=0.5
    [3] => Accept-Encoding: gzip, deflate, br
    [4] => X-Instagram-AJAX: 62d4dd0faf8e-hot
    [5] => X-IG-App-ID: 936619743392459
    [6] => X-IG-WWW-Claim: 0
    [7] => Content-Type: application/x-www-form-urlencoded
    [8] => X-Requested-With: XMLHttpRequest
    [9] => Origin: https://www.instagram.com
    [10] => Connection: keep-alive
    [11] => Referer: https://www.instagram.com/
    [12] => Pragma: no-cache
    [13] => Cache-Control: no-cache
    [16] => X-CSRFToken: 1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX
    [17] => Cookie: ig_cb=1; mid=XflzYgAAAAG5h0r8PPghWQ43I3u3; csrftoken=1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
)
Array
(
    [0] => Host: www.instagram.com
    [1] => Accept: */*
    [2] => Accept-Language: en-US,en;q=0.5
    [3] => Accept-Encoding: gzip, deflate, br
    [4] => X-Instagram-AJAX: 62d4dd0faf8e-hot
    [5] => X-IG-App-ID: 936619743392459
    [6] => X-IG-WWW-Claim: 0
    [7] => Content-Type: application/x-www-form-urlencoded
    [8] => X-Requested-With: XMLHttpRequest
    [9] => Origin: https://www.instagram.com
    [10] => Connection: keep-alive
    [11] => Referer: https://www.instagram.com/
    [12] => Pragma: no-cache
    [13] => Cache-Control: no-cache
    [16] => X-CSRFToken: 1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX
    [17] => Cookie: ig_cb=1; mid=XflzYgAAAAG5h0r8PPghWQ43I3u3; csrftoken=1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
    [18] => X-CSRFToken: qqrNK33XfZpJRVUqUmPhFO9jDf6YMQ0b
    [19] => Cookie: ig_cb=1; mid=XflzaQAAAAHGEuJcEdURke0RXCsH; csrftoken=qqrNK33XfZpJRVUqUmPhFO9jDf6YMQ0b; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
)
Array
(
    [0] => Host: www.instagram.com
    [1] => Accept: */*
    [2] => Accept-Language: en-US,en;q=0.5
    [3] => Accept-Encoding: gzip, deflate, br
    [4] => X-Instagram-AJAX: 62d4dd0faf8e-hot
    [5] => X-IG-App-ID: 936619743392459
    [6] => X-IG-WWW-Claim: 0
    [7] => Content-Type: application/x-www-form-urlencoded
    [8] => X-Requested-With: XMLHttpRequest
    [9] => Origin: https://www.instagram.com
    [10] => Connection: keep-alive
    [11] => Referer: https://www.instagram.com/
    [12] => Pragma: no-cache
    [13] => Cache-Control: no-cache
    [16] => X-CSRFToken: 1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX
    [17] => Cookie: ig_cb=1; mid=XflzYgAAAAG5h0r8PPghWQ43I3u3; csrftoken=1AQRCaPXTuzhtdnkUiEKu6t1Rs7o8eMX; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
    [18] => X-CSRFToken: qqrNK33XfZpJRVUqUmPhFO9jDf6YMQ0b
    [19] => Cookie: ig_cb=1; mid=XflzaQAAAAHGEuJcEdURke0RXCsH; csrftoken=qqrNK33XfZpJRVUqUmPhFO9jDf6YMQ0b; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
    [20] => X-CSRFToken: 2KdwmXuynin727Zuq97dLoFN49VJWFC2
    [21] => Cookie: ig_cb=1; mid=XflzbAAAAAEn4d6_ZW-SKDIIcxxz; csrftoken=2KdwmXuynin727Zuq97dLoFN49VJWFC2; ig_did=F3E4C932-1AC8-4918-8D75-04D82AF09F91; rur=FTW
)

```

**Comment by xosad** [comment]
Yes, thank you i think i've missed that out i'll fix it when i have time and push it.

**Comment by xosad** [comment]
Fixed 960222d8dd3c29625e9a1648f5fab9d1f4d6ffd2
